src/mondefense.cpp

```diff
@@ -41,3 +42,40 @@ void mdefense::zapback(monster *const m, Creature *const source, projectile cons
     }
     source->check_dead_state();
 }
+
+static int posp(int posc ) //Determines orientation of target from the source
+{
+    if (posc > 0){
+        return 1;
+    }
+    if (posc < 0){
+        return -1;
+    }
+    else {
+        return 0;
+    }
+}
+
+void mdefense::acidsplash(monster *const m, Creature *const source, projectile const* const proj)
+{
+    if( (proj) && ((rng(0, 40) > m->def_chance)) ) {
+        return; //Less likely for a projectile to deliver enough force
+    }
+    int dx = posp( source->posx() - m->posx() );
+    int dy = posp( source->posy() - m->posy() );
+    if (one_in(2)){
+        for (int i = 0; i < rng(3,5); i++) {
+            if (one_in(2)){
+                g->m.add_field(m->posx() + dx + rng(-1, 2),m->posy() + dy + rng(-1, 2), fd_acid, rng(2,3));
```

Comment:
I didn't intend for the larger chance to be symmetrical, I wanted it to be able to go behind the player if they were in melee range. I just realized though that that doesn't accomplish that. Doing `posx() + (dx *2) + rng(-1,1)` would be better